custom-recipes/timeseries-preparation-resampling/recipe.json

```diff
@@ -264,9 +264,49 @@
       ],
       "defaultValue": "clip"
     },
+    {
+      "name": "category_imputation_method",
+      "label": "Impute categorical data",
+      "description": "Used to fill in categorical values during interpolation and extrapolation",
+      "type": "SELECT",
+      "selectChoices": [
+        {
+          "value": "empty",
+          "label": "Empty"
+        },
+        {
+          "value": "constant",
+          "label": "Constant"
+        },
+        {
+          "value": "mode",
+          "label": "Most common"
+        },
+        {
+          "value": "clip",
+          "label": "Previous/next"
+        },
+        {
+          "value": "previous",
+          "label": "Previous"
+        },
+        {
+          "value": "next",
+          "label": "Next"
+        }
+      ],
+      "defaultValue": "empty"
+    },
+    {
+      "name": "category_constant_value",
+      "label": "Constant value",
+      "type": "STRING",
+      "defaultValue": "invalid",
```

Comment:
"invalid" seems to specific, I would set the default value with an empty string

Reply:
done in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/21/commits/07a221bfdbef07997291a13adf9c759c16ebce85